DentOS_Framework/DentOsTestbed/src/dent_os_testbed/constants.py

```diff
@@ -116,5 +117,6 @@
         'suite_functional_qos',
         'suite_functional_ipv4',
         'suite_functional_ipv6',
-        'suite_functional_l1']
+        'suite_functional_l1',
+        'suite_functional_devlink']
```

Comment:
Add comma at the end of line and move ] to next line. It will be easier to merge conflicts in future